third-party/cmake/BuildLuarocks.cmake

```diff
@@ -1,35 +1,89 @@
-option(USE_BUNDLED_BUSTED "Use the bundled version of busted to run tests." ON)
+# Luarocks recipe. Luarocks is only required when building Neovim, when
+# cross compiling we still want to build for the HOST system, whenever
+# writing a recipe than is mean for cross-compile, use the HOSTDEPS_* variables
+# instead of DEPS_* - check the main CMakeLists.txt for a list.
 
-if(USE_BUNDLED_LUAJIT)
-  list(APPEND LUAROCKS_OPTS
-    --with-lua=${DEPS_INSTALL_DIR}
-    --with-lua-include=${DEPS_INSTALL_DIR}/include/luajit-2.0)
+if(MSVC)
+  message(STATUS "Building busted in Windows is not supported (skipping)")
```

Comment:
I don't see the need to make it an option--I think you can just set the value and be done with it.